javascript/ql/experimental/adaptivethreatmodeling/lib/qlpack.yml

```diff
@@ -1,4 +1,5 @@
 name: codeql/javascript-experimental-atm-lib
+description: CodeQL libraries for the experimental ML-powered queries
```

Comment:
Do we want to include the term "Adaptive Threat Modeling (ATM)" in all of these descriptions?

Also, I wonder if the descriptions support html (or at least a subset of it).

Reply:
My understanding's that ATM is an internal-only term, and the customer-facing term is ML-powered queries.  Admittedly we're a little inconsistent about this given that we include "atm" in the pack name.